public/app/plugins/datasource/loki/backendResultTransformer.test.ts

```diff
@@ -31,7 +31,10 @@ const inputFrame: DataFrame = {
           json: true,
         },
       },
-      values: new ArrayVector(['{ "level": "info", "code": "41🌙" }', '{ "level": "error", "code": "41🌙" }']),
+      values: new ArrayVector([
+        { level: 'info', code: '41🌙' },
+        { level: 'error', code: '41🌙' },
+      ]),
```

Comment:
this unrelated test-case had a bug in it, i fixed it (switched from JSON_IN_STRING to javascript-object)